### DISCUSSION TITLE
The 'placeholder' attribute is unavailable using the latest @types/react package

### DISCUSSION THREAD

**Discussion by arusahni** [chakra-ui/chakra-ui#8594] [General]
### Description

With the most recent version of `@types/react` package [they removed the `placeholder` attribute from `HTMLAttributes`](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67170).

### Link to Reproduction

https://codesandbox.io/s/icy-http-9k2z43?file=/src/index.tsx

### Steps to reproduce

1. Navigate to the `index.tsx` file
2. Observe the type error for the `placeholder` attribute on the `SelectField` element

### Chakra UI Version

2.8.2

### Browser

Firefox (n/a)

### Operating System

- [ ] macOS
- [ ] Windows
- [X] Linux

**Comment by ghost** [comment]
This is happening for me across any all react packages or inputs in general regardless of chakra-ui. I'm very confused as to what we're supposed to do now. For me I am using Radix, but it doesn't really matter, and component I create that uses `extends React.HTMLAttributes<HTMLInputElement>` will no longer have the placeholder prop. It has been removed from HTMLAttributes per this issue: #67170

**Comment by algreasley** [comment]
I changed the base type for my component to be InputHTMLAttributes rather than HTMLAttributes .. if that helps
